banderwagon/precomp.go

```diff
@@ -139,11 +139,11 @@ func (pp *PrecompPoint) ScalarMul(scalar fr.Element, res *bandersnatch.PointProj
 				windowValue = (1 << pp.windowSize) - windowValue
 				if windowValue != 0 {
 					pNeg.Neg(&pp.windows[l*numWindowsInLimb+w][windowValue-1])
-					res.MixedAdd(res, &pNeg)
+					bandersnatch.ExtendedAddNormalized(res, res, &pNeg)
 				}
 				carry = 1
 			} else {
-				res.MixedAdd(res, &pp.windows[l*numWindowsInLimb+w][windowValue-1])
+				bandersnatch.ExtendedAddNormalized(res, res, &pp.windows[l*numWindowsInLimb+w][windowValue-1])
```

Comment:
L142 and L146 is where the real action changed.
We simply use the `Neg` and `ExtendedAddNormalized` methods while aggregating in the result.